npm-scripts.js

```diff
@@ -1,177 +1,209 @@
+			const { TscWatchClient } = require('tsc-watch/client');
 
-	case 'typescript:watch':
-	{
-		deleteNodeLib();
+			deleteNodeLib();
 
-		const { TscWatchClient } = require('tsc-watch/client');
-		const watch = new TscWatchClient();
+			const watch = new TscWatchClient();
 
-		watch.on('success', replaceVersion);
-		watch.start('--project', 'node', '--pretty');
+			watch.on('success', replaceVersion);
+			watch.start('--project', 'node', '--pretty');
 
-		break;
-	}
+			break;
+		}
 
-	case 'worker:build':
-	{
-		buildWorker();
+		case 'worker:build':
+		{
+			buildWorker();
 
-		break;
-	}
+			break;
+		}
 
-	case 'lint:node':
-	{
-		lintNode();
+		case 'worker:prebuild':
+		{
+			prebuildWorker();
 
-		break;
-	}
+			break;
+		}
 
-	case 'lint:worker':
-	{
-		lintWorker();
+		case 'lint:node':
+		{
+			lintNode();
 
-		break;
-	}
+			break;
+		}
 
-	case 'format:worker':
-	{
-		executeCmd(`${MAKE} format -C worker`);
+		case 'lint:worker':
+		{
+			lintWorker();
 
-		break;
-	}
+			break;
+		}
 
-	case 'test:node':
-	{
-		buildTypescript(/* force */ false);
-		replaceVersion();
-		testNode();
+		case 'format:worker':
+		{
+			executeCmd(`${MAKE} format -C worker`);
 
-		break;
-	}
+			break;
+		}
 
-	case 'test:worker':
-	{
-		testWorker();
+		case 'test:node':
+		{
+			buildTypescript(/* force */ false);
+			replaceVersion();
+			testNode();
 
-		break;
-	}
+			break;
+		}
 
-	case 'coverage:node':
-	{
-		buildTypescript(/* force */ false);
-		replaceVersion();
-		executeCmd('jest --coverage');
-		executeCmd('open-cli coverage/lcov-report/index.html');
+		case 'test:worker':
+		{
+			testWorker();
 
-		break;
-	}
+			break;
+		}
 
-	case 'install-deps:node':
-	{
-		installNodeDeps();
+		case 'coverage:node':
+		{
+			buildTypescript(/* force */ false);
+			replaceVersion();
+			executeCmd('jest --coverage');
+			executeCmd('open-cli coverage/lcov-report/index.html');
 
-		break;
-	}
+			break;
+		}
 
-	case 'install-clang-tools':
-	{
-		executeCmd('npm ci --prefix worker/scripts');
+		case 'install-deps:node':
+		{
+			installNodeDeps();
 
-		break;
-	}
+			break;
+		}
 
-	case 'release:check':
-	{
-		checkRelease();
+		case 'install-clang-tools':
+		{
+			executeCmd('npm ci --prefix worker/scripts');
 
-		break;
-	}
+			break;
+		}
 
-	case 'release':
-	{
-		checkRelease();
-		executeCmd(`git commit -am '${version}'`);
-		executeCmd(`git tag -a ${version} -m '${version}'`);
-		executeCmd(`git push origin v${MAYOR_VERSION}`);
-		executeCmd(`git push origin '${version}'`);
-		executeCmd('npm publish');
-
-		break;
-	}
+		case 'release:check':
+		{
+			checkRelease();
 
-	default:
-	{
-		throw new TypeError(`unknown task "${task}"`);
+			break;
+		}
+
+		case 'release':
+		{
+			checkRelease();
+			executeCmd(`git commit -am '${PKG.version}'`);
+			executeCmd(`git tag -a ${PKG.version} -m '${PKG.version}'`);
+			executeCmd(`git push origin v${MAYOR_VERSION}`);
+			executeCmd(`git push origin '${PKG.version}'`);
+			executeCmd('npm publish');
```

Comment:
```suggestion
			const { Octokit} = require('@octokit/rest');
			const octokit = new Octokit({ auth: process.env.GH_TOKEN });

			checkRelease();
			executeCmd(`git commit -am '${PKG.version}'`);
			executeCmd(`git tag -a ${PKG.version} -m '${PKG.version}'`);
			executeCmd(`git push origin v${MAYOR_VERSION}`);
			executeCmd(`git push origin '${PKG.version}'`);

			await octokit.repos.createRelease(
				{
					owner: 'integrated-cinematics',
					repo: 'mediasoup',
					name: version,
					tag_name: version,
					draft: false,
				}
			);
			
			executeCmd('npm publish');
```

This should automate the creation of a release for you.

Reply:
Amazing, thanks, will add it.